### DISCUSSION TITLE
Dashboard for Elsa Studio 3.2.0-preview.310

### DISCUSSION THREAD

**Discussion by rmanjusha** [elsa-workflows/elsa-core#5369] [Q&A]
How to enable Dashboard for Elsa Studio 3.2.0-preview.310

**Comment by glime-ai** [comment]
To enable the Dashboard for Elsa Studio version 3.2.0-preview.310, you should follow the standard setup instructions for Elsa Studio. Since you are referring to a preview version, ensure that you have included the preview feed URL in your `NuGet.config` as mentioned in the Elsa NuGet Packages documentation. Here are the general steps you would typically follow:

1. Create a new Blazor WebAssembly Application:
   ```
   dotnet new blazorwasm -n ElsaStudio
   ```

2. Install Elsa Studio NuGet Packages:
   ```
   dotnet add package Elsa.Studio --version 3.2.0-preview.310
   ```

3. Connect Elsa Studio to the Workflow Server:
   - Configure Elsa Studio to connect to the workflow server you have created.
   - Launch Elsa Studio and start designing your workflows.

Make sure to check the detailed guide on setting up Elsa Studio Blazor Application for any specific instructions related to the preview version you are using. If you encounter any issues with missing menus or features, ensure that the server version matches the client version and that you have correctly configured the connection between Elsa Studio and the Elsa Server.

If you are still facing issues, please provide more details about the problem, such as error messages or logs, so that I can assist you further.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
